docs/_articles/en/index.md

```diff
@@ -10,6 +10,13 @@ lang: en
 	<a href="./en/v3.x/whats-new-v3/">Check out what's new</a>
 </div>
 <br>
+<div class="slds-notify slds-notify_alert slds-theme_alert-texture slds-text-heading_small slds-theme_success" role="alert">
+  <span class="slds-assistive-text">success</span>
+    What do you think about Salesforce Code Analyzer?
+    &nbsp;&nbsp;
+    <a href="https://research.net/r/SalesforceCA">Share your feedback</a>
```

Comment:
Should we say, "Take the survey"? Or somehow bring in the word, "survey"? Because in all other references, we point them to logging an issue to give us feedback.

Reply:
Good point. Stand by for that.

Reply:
How about "Take our feedback survey"?

Reply:
Sounds good to me 👍

Reply:
@jfeingold35 @rmohan20 So I would consider this message a toast, so let's follow CCX toast conventions. 

My three suggestions:

"We released a new version of Salesforce Code Analyzer. Tell us what you think! Give Feedback."

OR

"We've improved Salesforce Code Analyzer. Tell us what you think! Give Feedback."

OR

"We're constantly improving Salesforce Code Analyzer. Tell us what you think! Give Feedback."

Link on Give Feedback-Can we include a Give Feedback button? To see more of what I'm talking about, check out the "spotlight on a toast" about 25% down this page: https://confluence.internal.salesforce.com/display/DOCTEAM/toast